pyproject.toml

```diff
@@ -30,6 +31,10 @@ ipykernel = "^5.3.4"
 ipython = "^7.16.1"
 pdoc3 = "^0.8.4"
 pandas = "^1.1.4"
+dask = {extras = ["distributed"], version = "^2021.2.0"}
+bokeh = "^2.2.3"
+jupyter-server-proxy = "^1.6.0"
```

Comment:
@galamit86 
don't see why we need jupyter libraries? Perhaps this is included accidentally whilst developing? Makes the image unnecessarily large.